app/models/task.rb

```diff
@@ -1,6 +1,25 @@
 class Task < ApplicationRecord
   belongs_to :user
 
+  validates :starts_at, presence: true
+  validates :ends_at, presence: true
+  validate :ended_after_started?
```

Comment:
`the task validate ended_after_started?` does not make sense as sentence.
but `the task validate ended_after_started` make sense.

and  does `ended_after_started?` need to return boolean?
in my opinion, the answer is `no`.